internal/commands/read_authzmodel_or_latest.go

```diff
@@ -0,0 +1,60 @@
+package commands
+
+import (
+	"context"
+	"errors"
+
+	grpc_ctxtags "github.com/grpc-ecosystem/go-grpc-middleware/tags"
+	"github.com/oklog/ulid/v2"
+
+	"github.com/openfga/openfga/pkg/typesystem"
+
+	serverErrors "github.com/openfga/openfga/pkg/server/errors"
+	"github.com/openfga/openfga/pkg/storage"
+)
+
+type ReadAuthorizationModelOrLatestQuery struct {
+	backend storage.AuthorizationModelReadBackend
+}
+
+// NewReadAuthorizationModelOrLatestQuery is a reusable command meant for other commands
+// where model ID parameter is optional.
+func NewReadAuthorizationModelOrLatestQuery(backend storage.AuthorizationModelReadBackend) *ReadAuthorizationModelOrLatestQuery {
```

Comment:
Existing commands in the `commands` package separate the RPC server definitions and their implementations. This seems to be breaking that mold. We're introducing a command that has no backing RPC and merely using it for shared functionality. 

It's not necessarily a problem, but I'm just pointing out that it breaks the status-quo of what we've been calling "commands". My fear is we're overloading code into commands and so the purpose of the commands in the first place is losing value.

Reply:
See https://github.com/openfga/openfga/issues/1382